Resources/Prototypes/_LostParadise/Datasets/nightmare.yml

```
@@ -0,0 +1,307 @@
+  - ГНОМЫ
+  - ОНИ ВСЕГДА СМОТРЯТ
+  - ОНИ ВСЕ ЗНАЮТ
+  - ТЕБЕ НЕ СКРЫТЬСЯ
+  - ГОВОРИ!
+  - ТЕБЕ НЕ СПРЯТАТЬСЯ!
+  - ТУМАН ГРЯДЕТ
+  - КОНЕЦ БЛИЗОК!
+  - Джек Потрошитель
+  - ЗАВЕДИ БЕНЗОПИЛУ
+  - утюг
+  - плита
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)